contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -591,3 +591,79 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.23, WithAnalyticsRate(0.23))
 	})
 }
+
+func TestIgnoredMethods(t *testing.T) {
```

Comment:
Can we have just one test `TestIgnoredMethods` like now, with two sub-tests? One `t.Run("unary", ...)` and the other `t.Run("stream", ...)`